setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = lsqecc
+version = 
+description = a Python package for ... 
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8
+url = https://latticesurgery.com/
+author = George Watkins, Alex Nguyen, Varun Seshardi, Keeland Watkins
+author_email = 
+license = 
+license_file = 
+classifiers =
+    Topic :: Scientific/Engineering
+    Development Status :: 3 - Alpha
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
```

Comment:
Just curious, have you tried running with 3.8 and 3.7. I've only tried 3.9

Reply:
Right now I am using 3.9 but I think I used it with 3.7 and 3.8 before. Might need to check again